bevy_proto_backend/src/proto/prototypes.rs

```diff
@@ -48,6 +60,30 @@ impl<'w, T: Prototypical> PrototypesMut<'w, T> {
         handle
     }
 
+    /// Load all the prototypes in the given directory.
+    ///
+    /// This will also store strong handles to the prototypes in order to keep them loaded.
+    ///
+    /// To load without automatically storing the handles, try using [`AssetServer::load_folder`].
+    pub fn load_folder<P: Into<Cow<'static, str>>>(&mut self, path: P) -> Result<Vec<HandleUntyped>, ProtoLoadError> {
+        let path = path.into();
+        let handles: Vec<_> = self.asset_server.load_folder(path.as_ref())?;
+
+        for handle in &handles {
+            let path = self.asset_server
+                .get_handle_path(handle)
+                .unwrap()
+                .path()
+                .to_str()
+                .ok_or(PathError::ConversionError)?
+                .to_string();
```

Comment:
Actually, what if we took in `P: Into<PathBuf>` and then stored the `PathBuf` in `ProtoStorage` instead of `Cow<'static, str>`?

This way we don't need to worry about conversions (we can remove the `ConversionError`) and it's a little more clear we're looking for paths to files and not just their ID or something.